test/manifest/encrypted.json

```diff
@@ -30,11 +33,21 @@
       "duration": 15,
       "key": {
         "method": "AES-128",
-        "uri": "https://priv.example.com/key.php?r=53"
+        "uri": "https://priv.example.com/key.php?r=53",
+        "iv": null
       },
       "uri": "http://media.example.com/fileSequence53-A.ts"
     },
     {
+      "duration": 14,
+      "key": {
+        "method": "AES-128",
+        "uri": "https://priv.example.com/key.php?r=54",
+        "iv": new Uint32Array([0, 0, 331, 3063767524])
```

Comment:
I'm not completely sure why this text is highlighted in red in github. A text encoding issue, maybe?